bin/ci-empty-queue-and-wait

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
```

Comment:
no `set -e` ?

It's preferred but not mandatory. If there are commands that may fail execution we should know which ones and how they might break. I see some `|| true` happening as well which typically mark such commands.

Given how the scripts are included and executed I suspect there might be some kind of flag inheritance happening but these scripts can also be run by themselves.